lang/fr/lang.php

```diff
@@ -169,15 +169,53 @@
             'default_level'          => 'Ce niveau est présélectionné pour l\'utilisateur.',
         ],
         'tabs'   => [
-            'description' => 'Description',
+            'description' => 'La description',
```

Comment:
In French, the defined article "LA" is not necessary in this context.

Reply:
@damsfx Sorry my French is really bad, I'm limited to counting to ten, saying bonjour and singing frere jacques, lol. Please translate all the english fields I added in to the file.

Reply:
@ayumihamsaki Don't apologize for your French, I'm myself trying to do my best in English! ;)

For update french lang file see : pull #18 .  
I will update the file as the project progresses.